uploaded_code/uploaded_code.ino

```diff
@@ -1,7 +1,15 @@
-//$ last work 23/Sep/23 [12:42 AM]
-// # version 5.4.9
-// # Release Note : Module can now read/write variables in SPIFFS
-
+<<<<<<< HEAD
+//$ last work 26/Sep/23 [01:30 AM]
+// # version 5.5.1
+// # Release Note : SMS bug fix
+// module was sending sms by setting sms_allowed to true it self: logic issue
+
+=======
+//$ last work 24/Sep/23 [08:45 AM]
+// # version 5.5.0
+// # Release Note : Module can now use and update variables of SPIFFS
+// Whatsapp functionality improved
+>>>>>>> 158fb5b5088d8bedf878c206ae0b5d5e69a2e332
```

Comment:
What is this bro? i know that this file is not useful but you also have to make sure what you upload. @mhamidjamil